src/index.html

```diff
@@ -8,9 +8,52 @@
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      method="get"
+      data-qa="big"
+    >
+      <div class="search-bar big-search-bar">
+        <label>
```

Comment:
This div is redundant. Move class to a label 
```suggestion
      <label class="search-bar big-search-bar">
```